website/blog/2022-08-12-model-timing.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Rooting out resource hogs with the Model Timing tab"
+description: "Monitoring large, complex projects can be difficult. When you're running 1,000+ models in a day, how do you know which of those consistently take the longest to run? In this article, Bennie Regenold and Barr Yaron show the benefits of the Model Timing tab in dbt Cloud."
+slug: model-timing-tab
+authors: [bennie_regenold, barr_yaron]
+tags: [analytics craft]
+hide_table_of_contents: false
+
+date: 2022-08-18
+is_featured: true
+---
+
+When running a job that has over 1,700 models, how do you know what a “good” runtime is? If the total process takes 3 hours, is that fantastic or terrible? While there are many possible answers depending on dataset size, complexity of modeling, and historical run times, the crux of the matter is normally “did you hit your SLAs”? However, in the cloud computing world where bills are based on usage, the question is really “did you hit your SLAs _and stay within budget_”? 
+
+Here at dbt Labs, we used the Model Timing tab in our internal analytics dbt project to help us identify inefficiencies in our incremental dbt Cloud job that eventually led to major financial savings, and a path forward for periodic improvement checks.
+
+<!--truncate-->
+
+## Your new best friend: The Model Timing tab
+
+The dbt Labs internal project is a beast. Our daily incremental dbt Cloud job runs 4x/day and invokes over 1,700 models.
+
+Sifting through this big run is easy with the [Model Timing](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab) tab in dbt Cloud. The `Model Timing` dashboard displays the model composition, order, and run time for every job run in dbt Cloud (for team and enterprise plans). The top 1% of model durations are automatically highlighted for quick reference. Here's an example of our incremental job before a fix was applied to our longest running model:
```

Comment:
This line feels a off tone - can we show how it makes life easier rather than telling?

Reply:
+ @barryaron 

@jasnonaz - The next few lines after this sentence show an example of how we were able to identify a problem model using the Model Timing tab. Is there something else you're thinking of for showing this?

Reply:
How about something like this @jasnonaz @bennieregenold7 :

The dbt Labs internal project is a beast! Our daily incremental dbt Cloud job runs 4x/day and invokes over 1,700 models. We like to sift through our dbt Cloud job using the [Model Timing](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-model-timing-tab) tab in dbt Cloud. The `Model Timing` dashboard displays the model composition, order, and run time for every job run in dbt Cloud (for team and enterprise plans). The top 1% of model durations are automatically highlighted, which makes it easy to find bottlenecks in our runs. Our longest running model stuck out like a sore thumb.  Here's an example of our incremental job before a fix was applied to our longest running model:

Reply:
Bumping this comment @jasnonaz + @bennieregenold7 + @KiraFuruichi

Reply:
Yeah I think this really encompasses what the model timing tab is all about in relation to a real-world project